src/components/Picker/Picker.js

```diff
@@ -219,12 +219,7 @@ class Picker extends PureComponent {
                                 },
                             ),
                         }}
-                        ref={(el) => {
-                            if (!_.isFunction(this.props.innerRef)) {
-                                return;
-                            }
-                            this.props.innerRef(el);
-                        }}
+                        focusRef={this.props.innerRef}
```

Comment:
@parasharrajat So this is the place where you would see the platform-dependent code inserted, right? Instead of `focusRef`, we'd switch whether to forward `this.props.innerRef` to `pickerProps.ref` or to `textInputProps.ref`?

Reply:
Okey, so I'll switch to that approach. I don't have anything against it. One more platform switch, but less changes in the fork.